server/routes/projects.js

```diff
@@ -115,6 +116,32 @@ router
 		}
 	});
 
+router
+	.route('/:projectId/comments/:commentId')
+	.delete(authenticateToken, async (req, res) => {
+		const { user } = req;
+		try {
+			user._id = isValidObjectId(user._id);
+			user.username = isValidUsername(user.username);
+			let projectId = isValidObjectId(req.params.projectId);
+			let commentId = isValidObjectId(req.params.commentId);
+			await projectsData.getProjectById(projectId);
+			await commentsData.getCommentById(commentId);
+			const commentObject = {
+				projectId,
+				commentId,
+			};
+			const removedComment = await commentsData.removeComment(
+				commentObject,
+				user
+			);
+			res.json({ removedComment });
```

Comment:
status code 204 (there is a status code object in `utils/index.js`
and with 204 you cannot return any response object, so it would be just res.status

Reply:
Done. Changed it.